sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -189,8 +187,11 @@ export class AppConfigurationClient {
    */
   getConfigurationSetting(
     key: string,
-    options?: GetConfigurationSettingOptions
+    options: GetConfigurationSettingOptions = {}
   ): Promise<GetConfigurationSettingResponse> {
+    options = { ...options };
+    options.label = options.label || undefined;
```

Comment:
Nit: the ||undefined isn’t needed since options.label would already be undefined.

Could you add a comment on why this is needed? If I came across this without context I’d be pretty confused!